detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/Nuget/Resolver/NugetLockFileResolver.cs

```diff
@@ -236,6 +259,28 @@ private bool ParseProjectFileDependencyGroupTokens(string input, string tokens,
             }
         }
 
+        private void saveDependency(String dependencyName, String dependencyVersion, DependencyResult result, PackageSetBuilder packageSetBuilder)
```

Comment:
As per [this comment,](https://github.com/blackducksoftware/detect-nuget-inspector/pull/4#discussion_r1284606738) repetitive for loop logic has been moved to functions.